src/atomate2/openmm/jobs/opls.py

```diff
@@ -0,0 +1,256 @@
+"""Utilities for working with the OPLS forcefield in OpenMM."""
```

Comment:
Currently this file is in the `jobs` folder but nothing here is a `job`. Perhaps this should be one directory up and called `utils.py`?

Reply:
`generate_openmm_interchange` is a job but I do agree some of this could be moved upstream to utils.